### DISCUSSION TITLE
Liman 1.7 Sürüm Duyurusu

### DISCUSSION THREAD

**Discussion by brnskn** [limanmys/core#148] [General (Genel)]
HAVELSAN tarafından geliştirilmekte olan Liman MYS 1.7 sürümü 8 Nisan 2021 tarihinde Havelsan Açıklab Ekibi tarafından duyuruldu.

Değişim Günlüğü
-Command injection ve XSS gibi güvenlik açıklarına karşı önlem alındı.
-Kullanıcı giriş ekranına Captcha koruması eklendi.
-Kullanıcı giriş ekranında yanlış parola denemeleri için throttle eklendi.
-Form validasyon hatalarının kullanıcıya detaylı gösterilmesi sağlandı.
-Hata düzeltmeleri yapıldı.
-Build operasyonları Github Actions'a taşındı.

Kurulum için:

https://rehber.liman.dev/liman-kurulum/

**Comment by aliorhun** [comment]
Açık Kaynak Kodlu örnek Liman MYS - Web Eklentisi:

https://github.com/MuratAlkanW/LimanMYS-Web

![dashboardTab](https://user-images.githubusercontent.com/260880/114077958-070f2b00-98b1-11eb-83fb-5cc3990ccddc.jpg)

**Comment by mertcelen** [comment]
![image](https://user-images.githubusercontent.com/9132682/114079104-6d487d80-98b2-11eb-93cc-8024184ee3e4.png)

tebrikler 👏🏻
